tf_files/aws/modules/vpc/cloud.tf

```diff
@@ -286,8 +286,10 @@ resource "aws_iam_instance_profile" "cluster_logging_cloudwatch" {
 resource "aws_route53_zone" "main" {
   name    = "internal.io"
   comment = "internal dns server for ${var.vpc_name}"
-  vpc_id  = "${aws_vpc.main.id}"
-
+  vpc {
```

Comment:
With this addition, I think we might be able to upgrade the terraform version we use